presto-tests/src/test/java/io/prestosql/tests/AbstractTestEngineOnlyQueries.java

```diff
@@ -109,6 +109,11 @@ public void testTimeLiterals()
         // TODO https://github.com/prestosql/presto/issues/37
         // TODO assertQuery(chicago, "SELECT TIME '3:04:05'");
         // TODO assertQuery(kathmandu, "SELECT TIME '3:04:05'");
+
+        assertThat(query("SELECT TIME '01:23:45.123'")).matches("SELECT TIME '01:23:45.123'");
```

Comment:
what does it mean that "X" matches "X" ?

Reply:
it means that the values was succesfully processed by the query runner (which wasn't the cxase before the fix)

maybe i should another add a test when the values differ and verify the assertion

Reply:
added more apparent sense to the code, ptal